src/context.h

```diff
@@ -342,6 +342,9 @@ class Context {
   /// simulation
   inline int n_specs(std::string impl) { return n_specs_[impl]; }
 
+  /// @return a const reference to the set of all agents in the simulation
+  inline const std::set<Agent*>& GetAgentList() const { return agent_list_; }
```

Comment:
I don't see this used anywhere but in tests.  Was it needed for something in particular? Or a nice-to-have?